### DISCUSSION TITLE
Hello Need Help For DSA

### DISCUSSION THREAD

**Discussion by CODGhost911** [community/community#124088] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Hello !
I am Current Get started with DSA,confused between Java and C++ ? which should i perfer?

**Comment by Atharvkote** [comment]
Well I had Same problem whiling get started with DSA!
I was confused in between c++ and java 
Then i did i little bit of research and found both are similar kinda Languages, doesn't matter which you choose ,i whould perfer java cause its Love ❤️ and practical reason java is little bit of beginner friendly Languages easy to understand then c++
THATS IT❤️

**Reply by Atharvkote** [reply]
Hope your DSA journey will be fantastic ,lots of love and respect 😉🔥⚡️

**Reply by CODGhost911** [reply]
Thank you boss🔥😁 ,for your guidance
